docs/source/integrations/pgvector.rst

```diff
@@ -0,0 +1,639 @@
+.. pgvector-integration:
+
+PostgreSQL Vector Search Integration
+=================================
+
+.. default-role:: code
+
+`PostgreSQL <https://www.postgresql.org>`_ is the world's most advanced open source database, and we've made it easy to use PostgreSQL's `vector search capabilities using PGVector <https://github.com/pgvector/pgvector>`_ on your computer vision data directly from FiftyOne!
```

Comment:
nit: our [docs style guide](https://github.com/voxel51/fiftyone/blob/develop/STYLE_GUIDE.md#documentation-style-guide) requests that maximum line length is 79 characters.

Unfortunately, enforcing this via [pre-commit hooks](https://github.com/voxel51/fiftyone/blob/09e7193a3953b03110a45fae42ebf154a0e1471f/.pre-commit-config.yaml#L8) causes bad things to happen (eg tables are broken, I believe), so the line length must be manually enforced via your `ENTER` key...